koku/koku/middleware.py

```diff
@@ -211,23 +252,24 @@ def process_request(self, request):  # noqa: C901
                 f'API: {request.path}{query_string}'
                 f' -- ACCOUNT: {account} USER: {username}'
             )
-            logger.info(stmt)
+            LOG.info(stmt)
             try:
                 customer = Customer.objects.filter(account_id=account).get()
             except Customer.DoesNotExist:
                 customer = IdentityHeaderMiddleware._create_customer(account)
+            except OperationalError as err:
+                LOG.error(err)
+                DB_CONNECTION_ERRORS_COUNTER.inc()
+                return HttpResponseFailedDependency()
```

Comment:
This is where we know it's a DB failure, so we could add the JSON body here.

Reply:
Message could be passed in like https://github.com/project-koku/koku/blob/master/koku/api/sources/view.py#L89-L93

Reply:
Postman result.  Looks good to me.
```{
    "errors": [
        {
            "detail": "Database unavailable. Error: could not translate host name \"db\" to address: Name or service not known\n",
            "status": 424,
            "title": "Failed Dependency"
        }
    ]
}```